js/core/css/caniuse.css

```diff
@@ -1,16 +1,14 @@
 /* container for stats */
-
 .caniuse-stats {
-  margin: 0;
-  padding: 0;
   display: flex;
   flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: baseline;
   cursor: pointer;
-  text-align: center;
 }
 
 button.caniuse-cell {
-  margin-top: 1px;
+  margin: 1px 1px 0 0;
```

Comment:
This was the main problem in iOS... buttons have margins in Safari for some reason.

Reply:
> buttons have margins in Safari for some reason

could `-webkit-appearance` have helped?

Reply:
Not sure, tbh. With *-appearance, you never know what you will get because it's platform specific (as far as I understand it).